src/server/service/kv.rs

```diff
@@ -72,13 +72,7 @@ impl<T: RaftStoreRouter + 'static, E: Engine> Service<T, E> {
         }
     }
 
-    fn send_fail_status<M>(
-        &self,
-        ctx: RpcContext<'_>,
-        sink: UnarySink<M>,
-        err: Error,
-        code: RpcStatusCode,
-    ) {
+    fn send_fail_status<M>(&self, ctx: RpcContext<'_>, sink: UnarySink<M>, err: Error, code: i32) {
```

Comment:
Not sure if we can do anything about it, but changing from a named type to i32 is a readability regression. If we have to do this, could you give the variable a more descriptive name or add a comment to describe what kind of a code it is and what the valid values are.

Reply:
I prefer to leave it to grpcio crates.

Reply:
@hunterlxt Do you want to give it a try? You can refer to the http crate for how to define a status code type.

Reply:
be glad to. So wrap our i32 into such this struct ? -> [refer](https://github.com/hyperium/http/blob/master/src/status.rs#L43)

Reply:
Yes. We should leave grpc-sys untouch and define the type in grpcio itself.